subsys/emds/Kconfig

```diff
@@ -68,8 +73,7 @@ config EMDS_FLASH_TIME_BASE_OVERHEAD_US
 	   worst case scenario, before starting storing entries. This value
 	   is dependent on the chip used, and should be checked against the chip
 	   datasheet.
-	   For RRAM-based flash drivers, the time is defined by the erasing block size.
-	   The default erasing block size is 4096 bytes.
+	   For RRAM-based persistent memory driver, use ERASEPROTECT and disregard the SOC_FLASH_NRF_PARTIAL_ERASE_MS parameter.
```

Comment:
`use ERASEPROTECT` what does it mean?

Reply:
See https://docs.nordicsemi.com/bundle/ps_nrf54L15/page/rramc.html#ariaid-title7

Reply:
Hm... it is not clear why is `ERASE PROTECT` required for emds? Seems this is up to the application whether it uses or doesn't.